lib/Expectation.lua

```diff
@@ -197,6 +197,33 @@ function Expectation:equal(otherValue)
 	return self
 end
 
+--[[
+	Assert that our expectation value is equal to another value within some limit, inclusive
+]]
+function Expectation:near(otherValue, limit)
```

Comment:
We should make sure that all the involved values are numbers, otherwise you'll get weird internal errors.